### PR TITLE
Fix Delete modals

### PR DESCRIPTION
**Pull request by ericnewcomer**
* Update to components where modals don't try to focus submit buttons
* Undo csrf token on hidden delete forms (unused)
* Fix flow label delete redirect (this was loading a page within the modal on success)

### CODE REVIEW

File: temba/flows/views.py [hunk 1]
```diff
@@ -2093,13 +2093,21 @@ class FlowLabelCRUDL(SmartCRUDL):
...
         submit_button_name = _("Delete")
 
+        def get_success_url(self):
+            return reverse("flows.flow_list")
+
+        def post(self, request, *args, **kwargs):
+            self.object = self.get_object()
+            self.object.delete()
+            return self.render_modal_response()
+
```
Comment (ericnewcomer) [inline temba/flows/views.py:2110]: Happened upon this bug which was isn't related to the current issue but is present in production, just following the model from `FlowCRUDL.Delete` here.

File: temba/msgs/views.py [hunk 2]
```diff
@@ -908,7 +908,7 @@ def derive_fields(self):
     class Delete(DependencyDeleteModal):
         cancel_url = "@msgs.msg_inbox"
         success_url = "@msgs.msg_inbox"
-        success_message = _("Your label has been deleted.")
+        success_message = ""
```
Comment (ericnewcomer) [inline temba/msgs/views.py:911]: Other views weren't pushing messages for these, so just making that consistent here.

Comment (rowanseymour) [reply temba/msgs/views.py:911]: It's kinda of a mixture at this point. The ones that extend `DependencyDeleteModal` mostly do, and it's still on my todo list to convert the rest. So let's decide whether we want these notifications.. I think I lean pro-toast

**Comment by rowanseymour** [comment]
Noticed the flow label delete doesn't close properly...

Captura de Pantalla 2021-06-15 a la(s) 09 47 08

File: temba/msgs/views.py (hunk 2 above)
Comment (ericnewcomer) [reply temba/msgs/views.py:911]: My problem with them is they only happen on page refresh and we don't always do that. Then they can stack up. I think notifications in general are good, but this way of doing it doesn't really work.

**Comment by ericnewcomer** [comment]
Did you see that with my changes in this PR? That's the same bug I happened upon testing that I mentioned here: https://github.com/nyaruka/rapidpro/pull/3358/files/c6d8a336b236f8d2cc2cbfe7afe1a79ed2019ee3#r651395560 which I believe is fixed in this PR.

**Comment by rowanseymour** [comment]
@ericnewcomer ah oops no, I thought that line was fixing deleting flows rather than flow labels

File: temba/msgs/views.py (hunk 2 above)
Comment (rowanseymour) [reply temba/msgs/views.py:911]: Ok well.. wanna pull all the `success_message`'s from the remaining modals based on `DependencyDeleteModal`? I don't hate reducing strings to translate...

Comment (ericnewcomer) [reply temba/msgs/views.py]: I reverted this for now, we can decide what we want in a different PR.

**Review by rowanseymour — APPROVED** [review]

**Pull request merged**